payload-schemas/schemas/common/github.schema.json

```diff
@@ -0,0 +1,90 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "common/github.schema.json",
+  "required": [
+    "login",
+    "id",
+    "node_id",
+    "url",
+    "repos_url",
+    "events_url",
+    "hooks_url",
+    "issues_url",
+    "members_url",
+    "public_members_url",
+    "avatar_url",
+    "description",
+    "name",
+    "company",
+    "blog",
+    "location",
+    "email",
+    "twitter_username",
+    "is_verified",
+    "has_organization_projects",
+    "has_repository_projects",
+    "public_repos",
+    "public_gists",
+    "followers",
+    "following",
+    "html_url",
+    "created_at",
+    "updated_at",
+    "type"
+  ],
+  "type": "object",
+  "properties": {
```

Comment:
I think we shouldn't go overboard with these constants as they could easily change and there's not much upside.
I think the main properties we should have as constants are `login`, `id`, `type`, and the url properties.

Reply:
I removed some constants from properties that could potentially change

Reply:
Cheers - theres still a few that I think could change (i.e `twitter_username` and `blog`), but you got the main ones (i.e `public_repos`) so this isn't a blocker for me any more :)